quarkus-app/src/main/resources/templates/AdminCampaignsResource/index.html

```
@@ -662,7 +662,7 @@ <h3>{item.channelLabel}</h3>
                       <input type="hidden" name="kind" value="{filters.kind}" />
                       <input type="hidden" name="channel" value="{filters.channel}" />
                       <input type="hidden" name="returnTo" value="channels" />
-                      <button type="submit" class="btn btn-secondary" id="campaignsPilotDisarmBtn">{copy.rolloutDisarmPilotActionLabel}</button>
+                      <button type="submit" class="btn btn--secondary" id="campaignsPilotDisarmBtn">{copy.rolloutDisarmPilotActionLabel}</button>
```

HTMLHint (1.9.2)

[error] The id value [ campaignsPilotDisarmBtn ] must be unique.

(id-unique)